src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -312,6 +312,19 @@ const schema = {
             },
           },
         },
+        separationLocation: {
+          type: 'object',
+          properties: {
+            separationLocationCode: {
+              type: 'string'
+            },
+            separationLocationName: {
+              type: 'string',
+              maxLength: 256,
+              pattern: "^([a-zA-Z0-9\/\-'.#,*()&][a-zA-Z0-9\/\-'.#,*()& ]?)*$",
```

Comment:
I'm reading this as no whitespace beyond spaces, no starting with space, no double spaces, and no bonkers punctuation, like underscores, carrots, double quotes, or mathematical punctuation.  Is that kind of the rule?  I take it the entire list of separation locations passes that test?

Reply:
I didn't check this regex against the predefined locations. We might want to double check @ndsprinkle!